### ISSUE TITLE
Craft3 fix for something fixed in Craft4 version

### ISSUE THREAD

**Issue by spAnser** [verbb/formie#1229]
### Describe the bug

Can we get this #1187 fixed in the Craft3 version of the plugin.

### Steps to reproduce

1. Add new Form
2. Configure reCaptcha integration
3. Query form with example above.

### Form settings

- Headless integration of Formie via Graphql

### Craft CMS version

3.7.62

### Plugin version

1.6.21

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by spAnser** [comment]
There is now an error if you try to query
```
integrations {
  settings
}
```

```
{
  "errors": [
    {
      "debugMessage": "Getting unknown property: verbb\\formie\\integrations\\captchas\\Recaptcha::scriptLoadingMethod",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "trace": [
        {
          "file": "/var/www/html/backend/vendor/verbb/formie/src/integrations/captchas/Recaptcha.php",
          "line": 252,
          "call": "yii\\base\\Component::__get('scriptLoadingMethod')"
        },
        {
          "file": "/var/www/html/backend/vendor/verbb/formie/src/gql/types/FormIntegrationsType.php",
          "line": 43,
          "call": "verbb\\formie\\integrations\\captchas\\Recaptcha::allowedGqlSettings()"
        },
```

**Comment by engram-design** [comment]
Should be fixed if you run `composer update` again.

**Comment by engram-design** [comment]
Fixed in [1.6.22](https://github.com/verbb/formie/releases/tag/1.6.22)
